stackanetes-services/cinder/cinder-db-sync.yml

```diff
@@ -0,0 +1,22 @@
+type: job
+name: cinder-db-sync
+image: cinder-api
+ceph_required: true
+command: bash /tmp/cinder-db-sync.sh
```

Comment:
why not jut place "cinder-manage db sync" in COMMAND env and get rid of /tmp/cinder-db-sync.sh file?